tests/test_normalize.py

```diff
@@ -8,13 +8,21 @@
 
 normalize_seq = partial(normalize, sequence=sequence)
 normalize_trim = partial(normalize_seq, mode=NormalizationMode.TRIMONLY)
+normalize_trim_no_shuffle = partial(normalize_seq, mode=None, trim=True)
+normalize_no_trim_no_shuffle = partial(normalize_seq, mode=None, trim=False)
 normalize_left = partial(normalize_seq, mode=NormalizationMode.LEFTSHUFFLE)
 normalize_right = partial(normalize_seq, mode=NormalizationMode.RIGHTSHUFFLE)
 normalize_expand = partial(normalize_seq, mode=NormalizationMode.EXPAND)
 normalize_vcf = partial(normalize_seq, mode=NormalizationMode.VCF)
+normalize_left_no_trim = partial(normalize_seq, mode=NormalizationMode.LEFTSHUFFLE, trim=False)
+normalize_right_no_trim = partial(normalize_seq, mode=NormalizationMode.RIGHTSHUFFLE, trim=False)
+normalize_expand_no_trim = partial(normalize_seq, mode=NormalizationMode.EXPAND, trim=False)
+normalize_vcf_no_trim = partial(normalize_seq, mode=NormalizationMode.VCF, trim=False)
 
 
-def test_trim():
+@pytest.mark.parametrize('normalize_trim', [normalize_trim, normalize_trim_no_shuffle])
```

Comment:
Nice code. One request: Don't overload the use of `normalize_trim` -- how about something else, like `normalize_fx` or `normer` or `larry`? Okay, not `larry`.